doc/man/8/rbd-nbd.rst

```diff
@@ -45,6 +45,12 @@ Options
 
    Forbid writes by other clients.
 
+.. option:: --timeout
+
+   Override device timeout. Linux kernel will default to a 30 seconds request timeout.
```

Comment:
... actually the original form was correct: s/seconds/second/ (second is used as an adjective instead of a noun).